src/analytics/controllers/helpers.py

```diff
@@ -36,7 +36,7 @@ def get_and_save_daily_measurements():
     return jsonify({'response': 'all daily measurements saved'}), 200
 
 
-@analytics_app.route('/api/v1/device/measurements/daily/update', methods =['GET'])
+@analytics_app.route('/api/v1/device/analytics/measurements/daily/update', methods =['GET'])
```

Comment:
looks like "analytics" should come before "device" here

Reply:
@danielogen thank you for the keen observation. This has been fixed.